lib/sequent/test/command_handler_helpers.rb

```diff
@@ -65,7 +70,7 @@ def commit_events(_, streams_with_events)
           end
         end
 
-        def given_events(streams_with_events)
+        def given_events(*streams_with_events)
```

Comment:
A fix for this is made in master. We like using just a sequence of events rather than event streams in specs.